express-notes-api/index.js

```diff
@@ -0,0 +1,94 @@
+const express = require('express');
+const fs = require('fs');
+const app = express();
+let data = {};
+
+fs.readFile('data.json', (err, dataFile) => {
+  if (err) {
+    console.error(err);
+  }
+  data = JSON.parse(dataFile);
+});
```

Comment:
It's ok to load your data when the server boots up, but it's much simpler to just `require()` it. That way, if there is a problem loading it, the server won't even start and give anyone false hope 😂 .

```js
const data = require('./data.json');
```